src/Components/ProductList/ProductList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import axios from 'axios';
+import OfferBox from 'components/OfferBox/OfferBox';
+
+const baseURL = 'https://61a7c977387ab200171d2ec9.mockapi.io/list';
+
+class ProductList extends React.Component {
+  state = {
+    products: []
+  }
+
+  componentDidMount() {
+    axios.get(baseURL)
+      .then(res => {
+        const products = res.data;
+        this.setState({ products });
+        console.log(this.state.products)
+      })
+      .catch(er => console.log(er))
+  }
+
+  render() {
+    return (
+      <ul>
+        {
+          this.state.products
+            .map(product => 
+              <OfferBox key={product.id} image={product.img} title={product.title} price={product.price} place={product.place} label={product.label}/>
+            )
```

Comment:
To jak już mówimy o dekonstrukcji propsów i innych rzeczy to tutaj 2 rady:
1. Każdy props w nowej linijce: 
```
<OfferBox
    key={id}
    image={img}
    title={title}
    price={price}
    place={place}
    label={label}
/>

Reply:
2. przy mapie można zrobić returna i przed returnem zdefiniować rzeczy np. 
`const {id, img, title, price, place, label} = product;`
i tak wygląda całość
 ```
this.state.products.map(product => {
    const {id, img, title, price, place, label} = product;
    return (
         <OfferBox
               key={id}
               image={img}
               title={title}
               price={price}
               place={place}
               label={label}
          />
     )
})